packages/twenty-server/src/engine/integrations/environment/environment-variables.ts

```diff
@@ -231,6 +231,18 @@ export class EnvironmentVariables {
   @IsString()
   SENTRY_DSN: string;
 
+  @ValidateIf(
+    (env) => env.EXCEPTION_HANDLER_DRIVER === ExceptionHandlerDriver.Sentry,
+  )
+  @IsString()
+  SENTRY_RELEASE: string;
+
+  @ValidateIf(
+    (env) => env.EXCEPTION_HANDLER_DRIVER === ExceptionHandlerDriver.Sentry,
+  )
+  @IsString()
+  SENTRY_ENVIRONMENT: string;
```

Comment:
I feel we should not introduce SENTRY_ENVIRONMENT. Maybe ENVIRONMENT, it can serve other purposes

Reply:
Maybe we could actually leverage process.env.NODE_ENV?

Reply:
Would it be ok to have process.env.NODE_ENV set to "main", "next" or "prod" ?